### ISSUE TITLE
[Zone/Quest/NPC] Darkmoon Island

### ISSUE THREAD

**Issue by TheSale** [WoWManiaUK/Blackwing-Lair#124]
Darkmoon Faire is in my opinion one of the best events in cataclysm, a lot of fun, in many aspects of the game,first is FUN then world bosses, fishing, questing, more FUN, pvp, farming, rewards, items, achievements, xmog, pets, mounts and maybe even more stuff and all that for all levels cross faction!

The thrill and nostalgy when I get ported in darkmoon were over the roof! I was so hyped and happy!
I bumped on few setbacks and issues and not working quests, but still I was so happy that this exist on wow-mania server.

let`s dive in issues that I managed to find in short time.

1. Items
Game tokens
https://www.wowhead.com/item=71083/darkmoon-game-token
They are found inside pouches sold by npc in darkmoon faire zone
https://www.wowhead.com/item=78899/pouch-o-tokens#contains
there are 2-3 npc`s in zone that sell this pouches for silver/gold there is scale of price to it from 2 silver to 2gold I believe and this should scale with reputation I believe.
The issue is that when I open pouches I dont get anything, I did tried all of them and not 1 single token.
amount that you get is determent by which pouch you bought, for 2 silver you should get 5 tokens I think.

Darkmoon Adventure Guide
http://wowwiki.wikia.com/wiki/Darkmoon_Adventurer%27s_Guide
this item works fine, I gotit when I commpleted quest
http://wowwiki.wikia.com/wiki/Quest:The_Darkmoon_Faire_(Alliance)
However if I delete or for what ever reason lose this item how would I get it back?
There should be few NPC that should give this item for free, I did not saw at any of them, maybe because I have item already with me, but I don`t want to delete item as I don`t know will I get it again, so this should be tested!

2. Quests
There are a lot quests in this zone, one time, daily, weekly and drop repetitive quests.
There should be 15 weekly quests and they are all profession related, 5 daily quests, 9 drop quests
and 1 quest that is done once during event time (test your strength).
All quests are for lvl 1 with an exception of drop quests, 2 quests are for max lvl 85,while other 7 quests are for lvl 15 and above.

So far I only completed and turn in First quest as I marked [working], second quest is still active and testing, weekly (professions) quests I did not tested because minimum requirement is 75+ skill for each profession and I have only 15 on mining. Drop items are obtained trough dungeons and pvp, and I cant test that yet (I am alone).
Fun quests Daily quests don`t work!
First time I tried I had no Game Tokens, so I tough that problem is there, however today I ask`d gm to give me game tokens and he did, but there is still same problem. Apparently game tokens were not an issue, the issue is that when I start quest nothing happen, I don`t get gun to shoot at the targets, I don`t get rings to throw on turtle, I don`t get mace to smash hoggers head, I can`t use cannon and can`t use racing cart to destroy zeppelins.

I would really like to see this fixed.
This is list of all quests

First quest one time quest [working[
http://wowwiki.wikia.com/wiki/Quest:The_Darkmoon_Faire

Repetitive every time darkmoon fair start [still testing]
http://wowwiki.wikia.com/wiki/Quest:Test_Your_Strength

Daily: [dont work]
http://wowwiki.wikia.com/wiki/Quest:Tonk_Commander
http://wowwiki.wikia.com/wiki/Quest:The_Humanoid_Cannonball
http://wowwiki.wikia.com/wiki/Quest:He_Shoots,_He_Scores!
http://wowwiki.wikia.com/wiki/Quest:Target:_Turtle
http://wowwiki.wikia.com/wiki/Quest:It%27s_Hammer_Time

Weekly: [did not tested]
Secondary professions 4
Archaeology http://wowwiki.wikia.com/wiki/Quest:Fun_for_the_Little_Ones
Fishing http://wowwiki.wikia.com/wiki/Quest:Spoilin%27_for_Salty_Sea_Dogs
Cooking http://wowwiki.wikia.com/wiki/Quest:Putting_the_Crunch_in_the_Frog
First Aid http://wowwiki.wikia.com/wiki/Quest:Putting_the_Carnies_Back_Together_Again

Primary professions: 
Alchemy http://wowwiki.wikia.com/wiki/Quest:A_Fizzy_Fusion
Blacksmithing http://wowwiki.wikia.com/wiki/Quest:Baby_Needs_Two_Pair_of_Shoes
Engeneering http://wowwiki.wikia.com/wiki/Quest:Talkin%27_Tonks
Enchanting http://wowwiki.wikia.com/wiki/Quest:Putting_Trash_to_Good_Use
Herbalism http://wowwiki.wikia.com/wiki/Quest:Herbs_for_Healing
Inscription http://wowwiki.wikia.com/wiki/Quest:Writing_the_Future
Jewelcrafting http://wowwiki.wikia.com/wiki/Quest:Keeping_the_Faire_Sparkling
Leatherworking http://wowwiki.wikia.com/wiki/Quest:Eyes_on_the_Prizes
Mining http://wowwiki.wikia.com/wiki/Quest:Rearm,_Reuse,_Recycle
Skinning http://wowwiki.wikia.com/wiki/Quest:Tan_My_Hide
Tailoring http://wowwiki.wikia.com/wiki/Quest:Banners,_Banners_Everywhere!

Drop Quests [Did not tested]
http://wowwiki.wikia.com/wiki/Quest:The_Master_Strategist
http://wowwiki.wikia.com/wiki/Quest:Tools_of_Divination
http://wowwiki.wikia.com/wiki/Quest:A_Curious_Crystal
http://wowwiki.wikia.com/wiki/Quest:An_Exotic_Egg
http://wowwiki.wikia.com/wiki/Quest:An_Intriguing_Grimoire
http://wowwiki.wikia.com/wiki/Quest:A_Wondrous_Weapon
http://wowwiki.wikia.com/wiki/Quest:A_Captured_Banner
http://wowwiki.wikia.com/wiki/Quest:The_Enemy%27s_Insignia
http://wowwiki.wikia.com/wiki/Quest:The_Captured_Journal

3. NPC [issue with npc interaction]
As I have played MOP on retail for like 20 days after cata, I am not sure if bosses (wolf and rabbit) in darkmoon were added in cata or pandaria, but it would be so fun to see them again. I did not saw them while I was exploring darkmoon island so I am maybe mistaken and they don`t exsist. :D

Interaction with NPC Tatia Brine <Fishing Supplies> located 52,88 on darkmoon island, do not want to open shop, or to have any interaction when I engage with her.

4. Fishing
This is in my opinion most important thing for darkmoon fair!
Low levels and fresh people can start and farm money right here! My lvl 20 on retail on new server earned around 1k gold in 1 day of leveling, fishing and selling!

Fishing school Shipwreck
https://www.wowhead.com/object=210216/shipwreck-debris
Crate from shipwreck
https://www.wowhead.com/item=78930/sealed-crate#fished-in
Shipwreck crate contain so much goodies and while fishing them I can get mount and pet aswell, and it is pretty decent chance like 5-10% chance for pet :)
also spawn time is insane I would say around 1 min, when there is 3 players fishing.
I know 100% that I was fishing with 2 more people and it was tight, almost exact time for fishing schools. when 2 persons fishing they will never run out of schools to fish from or to have to wait for spawn.

I explored 1/4 of seashore and not 1 single shipwreck were there this made me very sad. :(

5. PvP arena [wont let me enter the arena]
There is pvp arena similar as Gurubashi with the chest that spawn every 3 hours, but it has cage (like in gadgetzan tanaris) that dont want to let me to enter inside the arena, 
I wanted to try something more to enter there, but server went off.... 
but normal tries clicking on gate, did not let me enter, but chest was inside.

I would have more fun and found more things if server was longer online, but it come online around 16:20 and went off before 22h... Janitor, let me play :D

**Comment by giles1508** [comment]
something to add on to this,   darkmoon faire doesnt seem to be active when its supposed to be on the calendar,  i saw people there for like 1 or 2 days and then no one seemed to be able to get back there,  and the DMF is usually up for like 1.5 weeks on calendar  ,  and no one seems to know when its scripted to land on reign,   would it be possible to have a l;ook into this please to figure out firstly if its even spawning the event when its supposed to ?

**Comment by Kitzunu** [comment]
This is too big of a report.

Break it down and take one issue in one report, not a billion issues in one report

**Comment by TheSale** [comment]
well back then I didnt know how should reports be formed, and I put everything in 1 report, it looked as good idea at the moment.
After, I tried to make project, 
https://github.com/WoWManiaUK/Blackwing-Lair/projects/1
I tried to make "faster" that you dont have to open issue, but then github had some limitations on my idea, then I tried second project Speed up with existing issues, other people like it, more labels were added and it was success, but it was related with opened issues, 
When I realized that in order to darkmoon fair project get shape and look nice, I had to open every single issue separably, then I lost desire and give up on project and on issues of darkmoon fair.

But still I would like to see it works, and people have fun as I did on retail. :)
